src/renderer/lib/hooks/useSettings.ts

```diff
@@ -42,6 +42,14 @@ export const useMonthlySubfolders = () => {
   return [useMonthlySubfolders, setUseMonthlySubfolders] as const;
 };
 
+export const useJukeboxEnabled = () => {
```

Comment:
```suggestion
export const useJukebox = () => {
```
I think this is cleaner.